ocs_ci/helpers/virtctl.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from ocs_ci.framework import config
+
+# from ocs_ci.ocs import constants
+from ocs_ci.deployment.cnv import CNVInstaller
+from ocs_ci.helpers.github import get_asset_from_github
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+def get_virtctl_tool():
+    """
+    Download and install virtctl tool.
+
+    """
+    try:
+        virtctl_version = exec_cmd("virtctl version -c", silent=True)
+    except (CommandFailed, FileNotFoundError):
+        logger.info("virtctl tool is not available, installing it")
+        try:
+            cnv_obj = CNVInstaller()
+            cnv_obj.download_and_extract_virtctl_binary()
+            virtctl_version = exec_cmd("virtctl version -c")
+        except Exception:
+            virtctl_release_tag = config.ENV_DATA.get("virtctl_release_tag", "latest")
```

Comment:
maybe write an log message about why we fallback to this other mechanism to download from github with printing exception.

Reply:
Handled in [c67a518](https://github.com/red-hat-storage/ocs-ci/pull/10372/commits/c67a518938f82404bcd2b4b812c309edcae87511).